code/controllers/subsystem/SSjobs.dm

```diff
@@ -201,15 +204,9 @@ SUBSYSTEM_DEF(jobs)
 		if(player.mind && (job.title in player.mind.restricted_roles))
 			Debug("GRJ incompatible with antagonist role, Player: [player], Job: [job.title]")
 			continue
-		if(player.mind && (job.title in SSticker.mode.single_antag_positions))
-			if(!prob(probability_of_antag_role_restriction))
-				Debug("Failed probability of getting a second antagonist position in this job, Player: [player], Job:[job.title]")
-				continue
-			else
-				probability_of_antag_role_restriction /= 10
```

Comment:
Are you removing this and the same block in the proc below because it's redundant?

Reply:
redundant now because we directly check at giving them the role